### ISSUE TITLE
Inconsistent default config compare to spatie/laravel-permission

### ISSUE THREAD

**Issue by zorji** [Laravel-Backpack/PermissionManager#26]
I found that be behaviour of the following command generates different config.
```
php artisan vendor:publish --provider="Backpack\PermissionManager\PermissionManagerServiceProvider"
php artisan vendor:publish
```

They both generated /config/laravel-permission.php. The first one generated
```php
'user_has_permissions' => 'permission_users',
'user_has_roles' => 'role_users',
'role_has_permissions' => 'permission_roles',
```
while the second one generated
```php
'user_has_permissions' => 'user_has_permissions',
'user_has_roles' => 'user_has_roles',
'role_has_permissions' => 'role_has_permissions',
```

When I was trying this code in my development I was following the instructions and used the first command. However, in production code I used the second which generate all assets and configs.

I am just wondering is there a reason the name `permission_roles` is used instead of `role_has_permissions` in this project? If no, would you might I create a pull request to make them consistent?

**Comment by tabacitu** [comment]
Hi @zorji ,

Yes, we use a different laravel-permission config because the one provided by spatie/laravel-permission does not stick to the laravel standards, which backpack respects. In Laravel tables are usually named by the following rules:
- table name is plural, no capitals (ex: permissions);
- foreign keys are named by using the entity name in singular, underline id (ex: category_id);
- connection tables include both entity names in singular (ex: article_tag)

Spatie does their own thing there, and I had to pick one convention. So I chose the one that Taylor Otwell, Jeffrey Way and others use, not the one that Spatie does. I think they're the ones who make the most sense in this situation. [No ragrets](http://i.imgur.com/OG2hHwW.jpg).

Cheers!
